src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -79,6 +80,23 @@ public void satisfiesEqualsContract() throws IOException {
             ),
             "Content-Disposition: form-data; name=\"data\"; filename=\"a.bin\""
         );
+        final String boundary = "AaB02x";
```

Comment:
@yegor256  <code>RqMultipart.Fake</code> already calling inside constructor <code>RqMultipart.Base</code>. We are trying to test <code>RqMultipart.Base</code> but also we are trying to build inputs with using <code>RqMultipart.Base</code> . I believe that test input data should be independent from <code>RqMultipart.Base</code>. So in my opinion we should create test input request using <code>RqFake</code>